desktop/renderer-app/src/pages/BigClassPage/index.tsx

```diff
@@ -228,8 +229,18 @@ export const BigClassPage = observer<BigClassPageProps>(function BigClassPage()
         return (
             <>
                 <NetworkStatus networkQuality={classRoomStore.networkQuality} />
-                {!classRoomStore.isCreator && (
-                    <RoomInfo roomStatus={classRoomStore.roomStatus} roomType={RoomType.BigClass} />
+                {!classRoomStore.isCreator ? (
+                    <RoomInfo
+                        roomStatus={classRoomStore.roomStatus}
+                        roomType={classRoomStore.roomInfo?.roomType}
+                    />
+                ) : (
+                    classRoomStore.roomInfo?.beginTime && (
+                        <Timer
+                            roomStatus={classRoomStore.roomStatus}
+                            beginTime={classRoomStore.roomInfo.beginTime}
+                        />
+                    )
```

Comment:
FYI: https://eslint.org/docs/rules/no-negated-condition

I will add this rule to our project later.

```suggestion
                {classRoomStore.isCreator ? (
                    classRoomStore.roomInfo?.beginTime && (
                        <Timer
                            roomStatus={classRoomStore.roomStatus}
                            beginTime={classRoomStore.roomInfo.beginTime}
                        />
                    )
                ) : (
                    <RoomInfo
                        roomStatus={classRoomStore.roomStatus}
                        roomType={classRoomStore.roomInfo?.roomType}
                    />
```

Reply:
Sure. Glad to see your config.